SalesforceSDK/Salesforce.SDK.SmartStore/Util/Constants.cs

```diff
@@ -0,0 +1,195 @@
+        public const string TotalSize = "totalSize";
+        public const string RecentItems = "recentItems";
+        public const string SystemModstamp = "SystemModstamp";
+        public const string LastModifiedDate = "LastModifiedDate";
+
+        #region Salesforce object types.
+
+        public const string Account = "Account";
+        public const string Lead = "Lead";
+        public const string Case = "Case";
+        public const string Opportunity = "Opportunity";
+        public const string Task = "Task";
+        public const string Contact = "Contact";
+        public const string Campaign = "Campaign";
+        public const string User = "User";
+        public const string Group = "CollaborationGroup";
+        public const string Dashboard = "Dashboard";
+        public const string Content = "ContentDocument";
+        public const string ContentVersion = "ContentVersion";
+
+        #endregion
+
+        #region Salesforce object type field constants.
+
+        public const string KeyprefixField = "keyPrefix";
+        public const string NameField = "name";
+        public const string LabelField = "label";
+        public const string LabelpluralField = "labelPlural";
+        public const string FieldsField = "fields";
+        public const string LayoutableField = "layoutable";
+        public const string SearchableField = "searchable";
+        public const string HiddenField = "deprecatedAndHidden";
+        public const string NameFieldField = "nameField";
+        public const string NetworkidField = "NetworkId";
+        public const string NetworkscopeField = "NetworkScope";
+
+        #endregion
+
+        #region Metadata Manager constants
+
+        public const string MruCacheType = "recent_objects";
+        public const string MetadataCacheType = "metadata";
+        public const string LayoutCacheType = "layout";
+        public const string SmartScopesCacheKey = "smart_scopes";
+        public const string MruByObjectTypeCacheKey = "mru_for_%s";
+        public const string AllObjectsCacheKey = "all_objects";
+        public const string ObjectByTypeCacheKey = "object_info_%s";
+        public const string ObjectLayoutByTypeCacheKey = "object_layout_%s";
+
+        // Other constants.
+        public const string RecordTypeGlobal = "global";
+
+        #endregion
+
+        #region Salesforce object layout column field constants.
+
+        public const string LayoutNameField = "name";
+        public const string LayoutFieldField = "field";
+        public const string LayoutFormatField = "format";
+        public const string LayoutLabelField = "label";
+
+        #endregion
+
+        #region Salesforce object type layout field constants.
+
+        public const string LayoutLimitsField = "limitRows";
+        public const string LayoutColumnsField = "searchColumns";
+
+        #endregion
+
+        #region SyncState, SyncOptions, Status constants
```

Comment:
Did not want to add these changes in this PR as this PR is for solution restructuring. I am moving it in a separate file, but that will come after all the solution restructuring is done.